libsolidity/parsing/Parser.cpp

```diff
@@ -135,6 +135,15 @@ void Parser::parsePragmaVersion(SourceLocation const& _location, vector<Token> c
 			);
 }
 
+ASTPointer<FormalDocumentation> Parser::parseFormalDocumentation()
+{
+	ASTNodeFactory nodeFactory{*this};
+	nodeFactory.setLocation(m_scanner->currentCommentLocation());
```

Comment:
Here the location is set manually, and therefore https://github.com/ethereum/solidity/pull/7834/files#r372670260 is needed.